README.md

```diff
@@ -8,61 +8,53 @@ Needletail is a MIT-licensed, minimal-copying FASTA/FASTQ parser and _k_-mer pro
 The goal is to write a fast *and* well-tested set of functions that more specialized bioinformatics programs can use.
 Needletail's goal is to be as fast as the [readfq](https://github.com/lh3/readfq) C library at parsing FASTX files and much (i.e. 25 times) faster than equivalent Python implementations at _k_-mer counting.
 
-# Example
+## Example
```

Comment:
Is this intentional? Looks a bit like the whole block got commented out in an editor. (Referring not just to the header but to the `rust` code block below too)

Reply:
The # part is intentional, the commented out code not so much...